zoo-css-challenge/style.css

```diff
@@ -76,6 +85,8 @@ footer {
 }
 
 .introduction {
+  background-color: white;
+  line-height: 2.5;
```

Comment:
The instructions say that most of the text on the page is too close together, so maybe add line-height:2.5 to \* or body

Reply:
thank you Karen, I've now added line-height:2.5 to *